docs/tutorials/mobile/pose-detection.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Object detection and pose estimation with Yolov8
+description: Run the Yolov8 model on mobile (iOS and Android) with built-in pre and post processing to perform object detection and pose estimation
+image: /images/pose-thumbnail.png
+parent: Deploy on mobile
+grand_parent: Tutorials
+nav_order: 1
+---
+
+# Object detection and pose estimation on mobile with Yolov8
+
+Learn how to build and run ONNX models with built-in pre and post processing for object detection and pose detection.
+
+## Object detection with Yolov8
+
+You can find the full source code for the [Android](https://github.com/microsoft/onnxruntime-inference-examples/tree/main/mobile/examples/object_detection/ios) app in the ONNX Runtime inference examples repository.
```

Comment:
this links to the iOS app which is using ssd mobilenet.

Reply:
Oh yes, I will fix. I originally thought we had an iOS sample with Yolo